ui/shared/tabs/failureSummary/BugListItem.jsx

```diff
@@ -22,15 +31,35 @@ function BugListItem(props) {
           style={{ fontSize: '8px' }}
           type="button"
           onClick={() => addBug(bug, selectedJob)}
-          title="add to list of bugs to associate with all pinned jobs"
+          title="Add to list of bugs to associate with all pinned jobs"
         >
           <FontAwesomeIcon icon={faThumbtack} title="Select bug" />
         </Button>
       )}
       <span className="ml-1">i{bug.internal_id}</span>
       {!bug.id && (
-        <span className="ml-1">
-          {bug.summary} ({bug.occurrences} occurrences)
+        <span>
+          <span className="ml-1 mr-1" title="Number of recent classifications">
+            ({bug.occurrences} occurrences{' '}
+            <FontAwesomeIcon icon={faThumbtack} />)
+          </span>
+          {bug.summary}
+          {!bug.bugzilla_id && (
+            <Button
+              disabled={bug.occurrences < requiredInternalOccurrences}
```

Comment:
Don't disable but use always show as explained in the earlier comment and switch default positions for `>= requiredInternalOccurrences`. Also needs an update to the `title` below.